src/platform-includes/getting-started-primer/kotlin-multiplatform.mdx

```diff
@@ -1,13 +1,10 @@
 Sentry's Kotlin Multiplatform SDK builds on top of multiple Sentry SDKs, allowing developers to use the same codebase and share code between platforms while being able to integrate Sentry's features into their applications.
 
-## Overview of Features
```

Comment:
`l`: Generally speaking, I think these feature sections are useful, though I agree they shouldn't be the first thing users see when opening the platform docs. So food for thought: should we create a separate page for this? Maybe be a bit more descriptive? (totally fine as a separate PR)

Reply:
👍 will bring it up in the docs meeting